test.sh

```diff
@@ -8,6 +8,8 @@ pip install --user -e cupy/
 
 cd chainer
 
+pip install --user -e .[jenkins]
```

Comment:
Is it intentional to install Chainer twice?

Reply:
Fixed in 38e50893122d24470d236c0475feab23445ebae7.